contracts/upgradeable_contracts/U_ForeignBridge.sol

```diff
@@ -125,6 +143,7 @@ contract ForeignBridge is ERC677Receiver, BasicBridge {
     function deposit(address recipient, uint256 value, bytes32 transactionHash) external onlyValidator {
         bytes32 hashMsg = keccak256(recipient, value, transactionHash);
         bytes32 hashSender = keccak256(msg.sender, hashMsg);
+        require(withinHomeLimit(value));
```

Comment:
It is necessary to check limits and increment the daily limit by the value to check the limit in the next time properly. Please refer to the code in the `onTokenTransfer` method.

Reply:
Thanks, Done.